PRIVACY.md

```diff
@@ -0,0 +1,48 @@
+
+For the purposes of this notice, WE refers to the entity of Flagrow and its proprietors, FORUM refers to a Flarum website that has installed Flagrow Bazaar and YOU refers to its owner, APPLICATION refers to Flagrow Bazaar, WEBSITE refers to flagrow.io. 
+
+## Privacy Notice
+
+This privacy notice discloses the privacy practices for the APPLICATION. This privacy notice applies solely to information collected by this application and its associated WEBSITE. It will notify of the following:
+
+* What personally identifiable information is collected from YOU through the WEBSITE, how it is used and with whom it may be shared.
+* What choices are available to YOU regarding the use of your data.
+* The security procedures in place to protect the misuse of information YOU provide.
+* How YOU can correct any inaccuracies in the information.
+
+### Information Collection, Use, and Sharing
```

Comment:
Gonna hunt you until there's a regular spacing between titles and comments :smiling_imp: 

(always add a blank line under titles for example ?)